src/main/java/virushade/Ui.java

```diff
@@ -0,0 +1,152 @@
+package virushade;
+
+import virushade.tasks.TaskList;
+
+import java.util.Scanner;
+
+/**
+ * A utility class that deals with interactions with the user.
+ */
+public class Ui {
+    /**
+     * Prints a line break. Used after every interaction with Virushade.
+     */
+    private static void lineBreak() {
+        System.out.println("_____________________________________________________\n");
+    }
+
+    /**
+     * Prints the greeting.
+     */
+    private static void greet() {
+        System.out.println( "Hello from\n"
+                + "__      __ ( )                ____\n"
+                + "\\ \\    / /  _   ____  _   _  / __/\n"
+                + " \\ \\  / /  | | | ,__|| | | | \\ \\\n"
+                + "  \\ \\/ /   | | | |   | |_| | _) \\\n"
+                + "   \\__/    |_| |_|    \\__,_||___/\n"
+                + "What can <<Virushade>> do for you?\n");
+        lineBreak();
+    }
+
+    /**
+     * The echo function that Virushade uses. Echoes whatever words that Virushade is given.
+     * @param wordToEcho The word provided by the user for Virushade to echo.
+     */
+    public static void echo(String wordToEcho) {
+        System.out.println(wordToEcho);
+    }
+
+    /**
+     * Virushade first says goodbye before exiting.
+     */
+    private static void exit() {
+        System.out.println("Bye. Hope to see you again soon!");
+        lineBreak();
+        System.exit(0);
+    }
+
+    /**
+     * Handles unexpected inputs to Virushade
+     * @param str The input.
+     * @throws VirushadeException The response error message for the user.
+     */
+    private static void handleUnexpectedInputs(String str) throws VirushadeException {
+        if (str.startsWith("todo")) {
+            throw new VirushadeException("OOPS!!! The description of a todo task cannot be empty.");
+        } else if (str.startsWith("deadline")) {
+            throw new VirushadeException("OOPS!!! The description of a deadline task cannot be empty.");
+        } else if (str.startsWith("event")) {
+            throw new VirushadeException("OOPS!!! The description of an event task cannot be empty.");
+        } else if (str.startsWith("done")) {
+            throw new VirushadeException("OOPS!!! Please enter an integer after 'done'.");
+        } else if (str.startsWith("delete")) {
+            throw new VirushadeException("OOPS!!! Please enter an integer after 'delete'.");
+        } else if (str.startsWith("find")) {
+            throw new VirushadeException("OOPS!!! Please tell me what to find.");
+        } else {
+            throw new VirushadeException("OOPS!!! I'm sorry, but I don't know what that means :-(");
+        }
+    }
```

Comment:
You can consider using a switch statement here.

Reply:
Thanks for the input!!